### ISSUE TITLE
Invalid directive autocompletion

### ISSUE THREAD

**Issue by AkibAzmain** [swyddfa/esbonio#103]
When the extension shows autocomplete suggestions and I accept one of them, it automatically adds an extra space in front of the completion.

Example:

```
.. toc
      +-------------------+
      | toctree           |
      +-------------------+
      | ...               |
      +-------------------+
```

After accepting ``toctree``,

```
..  toctree::
```

**Comment by alcarney** [comment]
Thanks for taking the time to raise an issue!

The good news it that this should already be fixed as part of #101, it just hasn't made it into a release yet. 
Hopefully I can get one out in the next week or so :crossed_fingers:

**Comment by alcarney** [comment]
This should now be fixed with the latest release ([v0.5.0](https://github.com/swyddfa/esbonio/releases/tag/esbonio-lib-v0.5.0)) of the Language Sever, feel free to reopen this if it's still an issue.
